### PR TITLE
Research resources ui fixes

### PR DESCRIPTION
**Pull request by lizkrznarich**

### CODE REVIEW

**Comment by lizkrznarich** [comment]
[Research-resources with start but no end date sort to the bottom of the list chronologically](https://trello.com/c/cqaQTqqV/5834-research-resources-with-start-but-no-end-date-sort-to-the-bottom-of-the-list-chronologically)

**Comment by lizkrznarich** [comment]
[Research-resources remove show more function](https://trello.com/c/CEWUxiVg/5835-research-resources-remove-show-more-function)

**Comment by lizkrznarich** [comment]
[Research-resources show trash can on public view](https://trello.com/c/7XPMCAGA/5837-research-resources-show-trash-can-on-public-view)

File: orcid-web/src/main/webapp/static/javascript/ng1Orcid/require.js [hunk 1]
```diff
@@ -26,7 +26,6 @@ require("jquery");
 require("../jquery-migrate/1.3.0/jquery-migrate-1.3.0.min.js");
 require("../typeahead/0.9.3/typeahead.min.js");
 require("../script.js");
-require("../orcid.js");
```
Comment (leomendoza123) [inline orcid-web/src/main/webapp/static/javascript/ng1Orcid/require.js]: @lizkrznarich  Great we are removing this file. 

We can now remove it also from 

https://github.com/ORCID/ORCID-Source/blob/d7cd97b8be45081b59df704a4758bd669d532531/orcid-web/src/main/webapp/static/javascript/ng1Orcid/ci.webpack.config.js#L32
And all the other webpack files

I now that using a Regexp to map each script file by name on webpack the is not pretty but seems to be the only way.

**Pull request merged**